integration_tests/client/metrics/test_detection.py

```diff
@@ -153,8 +153,8 @@ def test_evaluate_detection(
         iou_thresholds_to_compute=[0.1, 0.6],
         iou_thresholds_to_return=[0.1, 0.6],
         filter_by=[
-            Annotation.labels.in_([Label(key="k1", value="v1")]),  # type: ignore - filter type error
```

Comment:
just checking: did you try doing a CTRL + F for "filter type error" to try and remove all of these ignore statements across all files?

Reply:
hmm i thought i had found them all haha

Reply:
you might have, I just wanted to double-check that you tried to search for them all